core/drivers/gpio/gpio.c

```diff
@@ -69,6 +69,12 @@ TEE_Result gpio_configure(struct gpio *gpio, enum gpio_flags flags)
 {
 	enum gpio_level value = GPIO_LEVEL_LOW;
 
+	assert(!gpio || (gpio->chip && gpio->chip->ops));
```

Comment:
It's strange to allow `gpio` to be NULL.

Reply:
Yep, it was from gpio_put but it does not apply here